support-frontend/package.json

```diff
@@ -152,7 +152,6 @@
     "@types/jsdom": "^21.1.1",
     "@types/lodash.debounce": "^4.0.6",
     "@types/react-dom": "^17.0.11",
-    "@types/react-redux": "^7.1.18",
```

Comment:
I think was during our bug fixing but am not sure why this hasn't complained at us